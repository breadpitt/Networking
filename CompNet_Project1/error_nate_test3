writeonlyuser
writepass
one_is_the_loneliest_file_that_youll_ever_know
