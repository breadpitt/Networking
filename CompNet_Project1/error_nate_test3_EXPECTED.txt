Received ACCESS_DENIED from server.
